src/components/Checkbox/Checkbox.stories.js

```diff
@@ -0,0 +1,27 @@
+import Checkbox from './Checkbox.vue';
+import mdx from './Checkbox.mdx';
+
+export default {
+  title: 'Components/Checkbox',
+  component: Checkbox,
+  parameters: {
+    docs: {
+      page: mdx
+    }
+  },
+};
+
+const Template = (args, {argTypes}) => ({
+  props: Object.keys(argTypes),
+  components: { Checkbox },
+  template: `
+    <checkbox 
+      v-bind="$props" 
+    ></checkbox>`,
+});
+
+export const Primary = Template.bind({});
+Primary.args = {
```

Comment:
Can we add a storybook example for multiple checkboxes to make it easier to test and see how to use it in that case?

Reply:
+1